src/main.zig

```diff
@@ -404,7 +404,8 @@ const usage_build_generic =
     \\  -fno-builtin              Disable implicit builtin knowledge of functions
     \\  -ffunction-sections       Places each function in a separate section
     \\  -fno-function-sections    All functions go into same section
-    \\  --strip                   Omit debug symbols
+    \\  --strip                   Force-enable debug symbols
+    \\  --no-strip                Force-disable debug symbols
```

Comment:
These descriptions are confusing to me. `--strip` force-enables debug symbols? It omits them so it rather "disables" them. I think this is better word choice:
```suggestion
    \\  --strip                   Omit debug symbols
    \\  --no-strip                Keep debug symbols
```